src/Filters/organisms/FilterBar.tsx

```diff
@@ -339,11 +459,12 @@ const FilterBar: React.FC<IFilterBar> = ({
           showLessText,
         }}
         searchFilters={createSearchers(searchersConfig, filtersValues, allowMultiFilter, handleSearchers)}
-        dropdownFilters={createDropdownFilters(dropdownsConfigRef.current, filtersValues, allowMultiFilter, handleSelected)}
+        dropdownFilters={createDropdownFilters(dropdownsConfigRef.current, filtersValues, allowMultiFilter, handleDropdownsSelected)}
+        datePickFilters={datePickersConfig ? createDatePickers(datePickersConfig, filtersValues, allowMultiFilter, handleDatePickers) : undefined}
```

Comment:
Is ‘datePick’ correct or should this be pickers for consistency?

Reply:
Thanks for pointing out, I missed that.